logging/config/cluster-logging-configuring.adoc

```diff
@@ -59,7 +59,7 @@ spec:
 ----
 You can configure the following for cluster logging:
 
-* You can place cluster logging into an unmanaged state that allows an administrator to assume full control of individual
+* You can place cluster logging in to an unmanaged state that allows an administrator to assume full control of individual
```

Comment:
I don't think "into" needs to be separated in this case. Depends on whether it is referring to "cluster logging" or "cluster log in" as a concept, if that makes sense.

Reply:
You're right - it doesn't need to be separated.